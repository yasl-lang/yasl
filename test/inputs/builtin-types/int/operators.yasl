echo 2 ** 4
echo 2 ** -1
echo -2 ** -1

echo 5 * 6
echo 5 / 2
echo 6 // 2
echo 5 % 2

echo 10 + 12
echo 3 - 2
