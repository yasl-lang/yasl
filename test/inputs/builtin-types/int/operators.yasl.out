16
0.5
-0.5
30
2.5
3
1
22
1
